CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+# Welcome to Fleet Telemetry contributing guide <!-- omit in toc -->
+
+Thank you for investing your time in contributing to our project!
```

Comment:
`Thank you for investing your time contributing to the project!` (remove "in" and "our")